scale/scheduler/manager.py

```diff
@@ -13,7 +13,7 @@
 from util.active_warnings import ActiveError, ActiveWarning
 from util.parse import datetime_to_string
 
-CLEANUP_WARN_THRESHOLD = datetime.timedelta(hours=3)
+CLEANUP_WARN_THRESHOLD = datetime.timedelta(hours=1)
```

Comment:
Good change. Clean up those warnings. We may want to make it even shorter. 👍

Reply:
Even better, we may want to look into reporting job types rather than a huge list of job ids for troubleshooting.